### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LLouver**
加入链接
[【PyTorch】repeat_interleave()方法详解_torch获取某个维度重复的数据-CSDN博客](https://blog.csdn.net/weixin_45261707/article/details/119187799)
[pytorch中nn.Dropout的使用技巧_在pytorch框架下使用dropout只需要在模型中添加 nn.dropout(0.2)-CSDN博客](https://blog.csdn.net/leviopku/article/details/120786990)
[MAE 自监督算法介绍和基于 EasyCV 的复现_mae算法原理-CSDN博客](https://blog.csdn.net/yunqiinsight/article/details/124877543)
[通俗讲解pytorch中nn.Embedding原理及使用 - 简书](https://www.jianshu.com/p/63e7acc5e890)
[NLP中Mask全解_glm attention mask-CSDN博客](https://blog.csdn.net/songyunli1111/article/details/108023139)
[Prompt-Tuning——深度解读一种新的微调范式_prompt tuning-CSDN博客](https://blog.csdn.net/qq_36426650/article/details/120607050)

### CODE REVIEW

**Comment by LLouver** [comment]
[Git - 初次运行 Git 前的配置](https://git-scm.com/book/zh/v2/%E8%B5%B7%E6%AD%A5-%E5%88%9D%E6%AC%A1%E8%BF%90%E8%A1%8C-Git-%E5%89%8D%E7%9A%84%E9%85%8D%E7%BD%AE)
[joblib](https://blog.csdn.net/qq_41185868/article/details/108278072)
[Python 实现秒表功能 | 菜鸟教程](https://www.runoob.com/python3/python-simplestopwatch.html)
[torch.Tensor - PyTorch中文文档](https://pytorch-cn.readthedocs.io/zh/latest/package_references/Tensor/)
[Pytorch实现两层神经网络 - 当康 - 博客园](https://www.cnblogs.com/luedong/p/14492361.html)

**Comment by LLouver** [comment]
[什么是张量（Tensor）-CSDN博客](https://blog.csdn.net/weixin_42259833/article/details/124766853?ops_request_misc=%257B%2522request%255Fid%2522%253A%2522172006127716777224458282%2522%252C%2522scm%2522%253A%252220140713.130102334..%2522%257D&request_id=172006127716777224458282&biz_id=0&utm_medium=distribute.pc_search_result.none-task-blog-2~all~top_positive~default-1-124766853-null-null.142^v100^pc_search_result_base1&utm_term=%E5%BC%A0%E9%87%8F&spm=1018.2226.3001.4187)
[多元时序预测：独立预测 or 联合预测？-腾讯云开发者社区-腾讯云](https://cloud.tencent.com/developer/article/2313300)
[ML-NLP/NLP/16.4 textRNN & textCNN at master · NLP-LOVE/ML-NLP](https://github.com/NLP-LOVE/ML-NLP/tree/master/NLP/16.4%20textRNN%20%26%20textCNN)
[【论文下饭】PatchTST中的channel-independence-CSDN博客](https://blog.csdn.net/LittleSeedling/article/details/130808893)
[3d卷积的cnn网络 cnn卷积运算_mob64ca13fb1f2e的技术博客_51CTO博客](https://blog.51cto.com/u_16213595/10166901)
[3D卷积，代码实现 - 吴建明wujianming - 博客园](https://www.cnblogs.com/wujianming-110117/p/15009170.html)
[Transformer解析与tensorflow代码解读 - 不著人间风雨门 - 博客园](https://www.cnblogs.com/zhouxiaosong/p/11032431.html)
[Transformer——patch embedding代码-CSDN博客](https://blog.csdn.net/qq_42740834/article/details/124994344)
[论文](https://arxiv.org/abs/1810.04805v2)
[深度学习的优化器（各类 optimizer 的原理、优缺点及数学推导） - FromL77 - 博客园](https://www.cnblogs.com/froml77/p/14956375.html)
[CNN中的参数解释及计算](https://flat2010.github.io/2018/06/15/%E6%89%8B%E7%AE%97CNN%E4%B8%AD%E7%9A%84%E5%8F%82%E6%95%B0/)
[nn.LayerNorm的实现及原理-CSDN博客](https://blog.csdn.net/weixin_41978699/article/details/122778085)
[自监督学习之掩码自动编码器(Masked Autoencoders, MAE)——音频识别方面_掩码自编码器-CSDN博客](https://blog.csdn.net/pk296256948/article/details/128666880)
[Pytorch之Embedding与Linear的爱恨纠葛 - 奥辰 - 博客园](https://www.cnblogs.com/chenhuabin/p/17117992.html)
[【Pytorch】nn.Dropout的用法-CSDN博客](https://blog.csdn.net/weixin_47050107/article/details/122722516)
[torch.sort()和torch.argsort()简要介绍-CSDN博客](https://blog.csdn.net/weixin_45335726/article/details/122362383)
[详解 Pytorch.random-CSDN博客](https://blog.csdn.net/ViatorSun/article/details/110109694)
[图解torch.gather()的用法_torch.gather 三维使用方法-CSDN博客](https://blog.csdn.net/iteapoy/article/details/106203954)
[大语言模型的预训练[3]之Prompt Learning：Prompt Engineering、Answer engineering详解-腾讯云开发者社区-腾讯云](https://cloud.tencent.com/developer/article/2303277)
[五万字综述！Prompt-Tuning：深度解读一种新的微调范式 - 知乎](https://zhuanlan.zhihu.com/p/618871247)
